cea/analysis/sensitivity/sensitivity_demand_analyze.py

```diff
@@ -142,22 +164,36 @@ def read_results(samples_folder, samples_count):
                           `sensitivity_demand_count.py` to calculate this for a given samples folder.
     :type samples_count: int
 
+    :param output_parameter: output parameter of the simualation e.g., Qhsf, Ef
+    :type samples_count: str
+
+    :param output_parameter: output parameter of the simualation e.g., Qhsf, Ef
+    :type samples_count: str
+
     RETURNS
     -------
 
-    :returns: A list of DataFram objects representing each result of a simulation.
+    :returns: matrix of lenght mxn where m: samples_count, n: buildings, content: result for output_parameter
     :rtype: list of DataFrame
 
     INPUT / OUTPUT FILES
     --------------------
 
     - `$samples_folder/result.$i.csv` for i in range(samples_count)
     """
-    results = []
-    for i in range(samples_count):
-        result_file = os.path.join(samples_folder, 'result.%i.csv' % i)
-        df = pd.read_csv(result_file)
-        results.append(df)
+    iterable_samples_count = range(samples_count)
```

Comment:
I thought it was iterable of the form [0......n]. the variable is a vector not an integer.